docs/modules/ROOT/pages/compatibility.adoc

```diff
@@ -14,7 +14,8 @@ NOTE: RuboCop might be working with other Ruby implementations as well, but it's
 == Support Matrix
 
 RuboCop generally aims to follow MRI's own support policy - meaning RuboCop would support all officially supported MRI releases.footnote:[Typically the last 3 releases.] To give people extra time for a smooth transition, we've customarily provided support for about one year after EOL of MRI version.
-This means that if Ruby 2.7 reaches its EOL in Spring 2023, it would be supported by RuboCop (at least) until Spring 2024.footnote:[At the core team's discretion this policy might be waived aside for MRI releases causing significant maintenance overhead.]
+Although there are major version incompatibilities between Ruby 2.7 and Ruby 3.0, and the latest stable version of Ruby on Rails, 7.1, still supports Ruby 2.7. So, dropping support for Ruby 2.7 from RuboCop might be premature for the market at this stage.
```

Comment:
I'd rather convert this to a note, that we've decided to extend support for Ruby 2.7 given it's the last release of the 2.x series and we plan to support it until it's supported by supported Rails releases (or something along those lines).

Reply:
I get it. I added `NOTE:` to the text. Should the text be adjusted as well?